src/NuGet.Core/NuGet.Common/ComparisonUtility.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Common
+{
+    public static class ComparisonUtility
+    {
+        public static StringComparer FrameworkReferenceNameComparer = StringComparer.OrdinalIgnoreCase;
```

Comment:
As-is, this is a writable field.  It should be read-only.

You have two options:
1. convert into a read-only property
2. convert into a read-only field

I prefer option 2 as it's simpler.